### DISCUSSION TITLE
Hi , I applied for GitHub copilot . (edited)

### DISCUSSION THREAD

**Discussion by Anvisimi** [community/community#136628] [GitHub Education]
### Hi everyone,

_Moderators Edit_
>  First I was asked to submit the following : Hey Simran Arora,  Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Aug 22, 2024 for @Anvisimi.  Please secure your GitHub account with two-factor authentication. You may need to log out and log back in to GitHub before reapplying. Please complete your GitHub billing information with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before reapplying. If you have only a single legal name, enter it in both the first and last name fields. Please use your device camera to submit academic affiliation documents and reapply

Now it says : Photo has previously been used. Please upload a new photo.
I just have one id card how to upload a new photo

**Comment by ferid333** [comment]
It sounds like GitHub is asking you to provide a new photo of your ID card because the one you submitted before has already been used. Since you only have one ID card, here’s what you can do:

1. **Take a New Photo:** Try taking a new photo of your ID card with your phone or camera. Make sure it's clear and shows all the necessary details.

2. **Check for Errors:** Ensure that the photo you take is well-lit and not blurry.

3. **Upload the New Photo:** Go back to the GitHub application page and upload the new photo of your ID card.

If you’re still having trouble, you might want to contact **GitHub support** for more help.
